decisions/wc_dark_cult_decisions.txt

```diff
@@ -637,11 +637,29 @@ decisions = {
 			prisoner = no
 			is_inaccessible_trigger = no
 			independent = yes
-			
+
 			government = black_pact_government
-			
+
 			religion_authority = 0.75
 			num_of_holy_sites = 4
+			trigger_if = {
+				limit = { cthun_offmap_decisions_enabled_trigger = yes }
+				offmap_cthun = { has_policy = old_god_working }
+				b_ahnqiraj = { holder_scope = { character = ROOT } }
+			}
+			trigger_else_if = {
+				limit = { nzoth_offmap_decisions_enabled_trigger = yes }
+				offmap_nzoth = { has_policy = old_god_working }
+				b_sagehold = { holder_scope = { character = ROOT } }
+			}
+			trigger_else_if = {
+				limit = { yogg_saron_offmap_decisions_enabled_trigger = yes }
+				offmap_yoggsaron = { has_policy = old_god_working }
+				b_ulduar = { holder_scope = { character = ROOT } }
+			}
+			trigger_else = {
+				always = no
+			}
```

Comment:
It's not clear what you need to do. Mention that you can't reborn the Black Empire if you don't follow any Old God.